ci/build_wheel_python.sh

```diff
@@ -24,6 +24,7 @@ cd "${package_dir}"
 echo "libkvikio-${RAPIDS_PY_CUDA_SUFFIX} @ file://$(echo ${CPP_WHEELHOUSE}/libkvikio_*.whl)" > ./constraints.txt
 
 PIP_CONSTRAINT="${PWD}/constraints.txt" \
+SKBUILD_CMAKE_ARGS="-DFIND_KVIKIO_CPP=ON;-DCMAKE_INSTALL_INCLUDEDIR=kvikio/include;-DCMAKE_INSTALL_LIBDIR=kvikio/lib64" \
```

Comment:
Why do we need to specify the include and lib install dirs for the Python package? What gets installed to these directories, and why isn't it part of the libkvikio package instead?

Reply:
Looks like it's currently just installing `nvcomp` and `gdeflate`. I'll see if there's some way to not have these present in the wheel.

Reply:
I've prevented `nvcomp` from being installed and removed these arguments.

Reply:
> I think installing libnvcomp*.so while avoiding installing the headers is the right choice because we don't want to make our wheels provide a complete installation of nvcomp that consumers could try to use.

That's what's currently happening here. `libnvcomp*.so` is located in the `kvikio/_lib` directory, and no headers are installed.

Reply:
Oh good right we do this correctly [here](https://github.com/rapidsai/kvikio/blob/branch-24.10/python/kvikio/CMakeLists.txt#L58). I forgot that I walked Michael through this in #369. This looks good, then.